framework/source/class/qx/ui/layout/Grid.js

```diff
@@ -108,6 +116,10 @@ qx.Class.define("qx.ui.layout.Grid",
     if (spacingY) {
       this.setSpacingY(spacingY);
     }
+    
+    if(allowGrowSpannedCellWidth === true || allowGrowSpannedCellWidth === false) {
```

Comment:
If that parameter is removed, this block would be removed as well. If the parameter remains, then a more common paradigm might be `if (typeof allowGrowSpannedCellWidth == "boolean")`